ViAn/GUI/mainwindow.cpp

```diff
@@ -389,3 +399,84 @@ bool MainWindow::eventFilter(QObject *obj, QEvent *event) {
     }
     return false;
 }
+/**
+ * @brief MainWindow::prepare_menu
+ * @param pos
+ * Creates context menu on right-click in tree view
+ */
+void MainWindow::prepare_menu(const QPoint & pos) {
+    QTreeWidget *tree = ui->ProjectTree;
+
+    MyQTreeWidgetItem *item = (MyQTreeWidgetItem*)tree->itemAt( pos );
+
+    QMenu menu(this);
+
+    if(item == nullptr) {
+
+    } else if(item->type == TYPE::PROJECT) {
+        set_selected_project(item);
+        QAction *addVideo = new QAction(QIcon(""), tr("&Add video"), this);
+        addVideo->setStatusTip(tr("Add video"));
+        menu.addAction(addVideo);
+        connect(addVideo, SIGNAL(triggered()), this, SLOT(add_video()));
+    } else if(item->type == TYPE::VIDEO) {
+        selectedVideo = item;
+        QAction *loadVideo = new QAction(QIcon(""), tr("&Play video"), this);
+        loadVideo->setStatusTip(tr("Play video"));
+        menu.addAction(loadVideo);
+        connect(loadVideo, SIGNAL(triggered()), this, SLOT(play_video()));
+    }
+
+
+
+    QPoint pt(pos);
+    menu.exec( tree->mapToGlobal(pos) );
+}
```

Comment:
its very small change but look through the code for these kind of empty lines. Some empty lines are making it easier to read but you don't need more than one new line